tests/integration_tests/functional/test_vhost_user_block.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests for vhost-user-block device."""
+
+import os
+import subprocess
+import time
+
+from framework import utils
+
+VHOST_USER_SOCKET = "/vub.socket"
+
+
+def spawn_vhost_user_backend(vm, host_mem_path):
+    """Spawn vhost-user-block backend."""
+
+    uid = vm.jailer.uid
+    gid = vm.jailer.gid
+
+    sp = f"{vm.chroot()}{VHOST_USER_SOCKET}"
+    args = ["vhost-user-blk", "-s", sp, "-b", host_mem_path, "-r"]
+    proc = subprocess.Popen(args)
+
+    time.sleep(1)
+    if proc is None or proc.poll() is not None:
+        print("vub is not running")
+
+    with utils.chroot(vm.chroot()):
+        # The backend will create the socket path with root rights.
+        # Change rights to the jailer's.
+        os.chown(VHOST_USER_SOCKET, uid, gid)
+    return proc
+
+
+def test_vhost_user_block(microvm_factory, guest_kernel, rootfs_ubuntu_22):
+    """
+    This test simply tries to boot a VM with
+    vhost-user-block as a root device.
+    """
+
+    vm = microvm_factory.build(guest_kernel, None, monitor_memory=False)
+
+    vm.jailer.extra_args.update({"no-seccomp": None})
```

Comment:
we should update the seccomp filters instead. Can use the change in https://github.com/firecracker-microvm/firecracker/pull/4069/commits/65be7ae66719f6ed836d1dafed25a3b10a58b8c5 .

Reply:
done